site/content/en/latest/install/install-yaml.md

```diff
@@ -11,23 +11,19 @@ installation, it is recommended that you use helm.
 
 ## Before you begin
 
-Envoy Gateway is designed to run in Kubernetes for production. The most essential requirements are:
-
-* Kubernetes 1.28 or later
-* The `kubectl` command-line tool
-
 {{% alert title="Compatibility Matrix" color="warning" %}}
 Refer to the [Version Compatibility Matrix](/news/releases/matrix) to learn more.
 {{% /alert %}}
 
-## Install with YAML
-
-Envoy Gateway is typically deployed to Kubernetes from the command line. If you don't have Kubernetes, you should use `kind` to create one.
+Envoy Gateway is typically deployed in a Kubernetes cluster.
```

Comment:
Not a must, can you make this into boilerplates?

Reply:
Let's keep it for now?  It's just two simple sentences.  If this becomes more complex in the future, then we can move it to a bolierplate.